ios/MindloggerMobile.xcodeproj/project.pbxproj

```diff
@@ -102,7 +102,7 @@
 		4E9EC54929BA279C002F69A5 /* MaterialIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 4E9EC54629BA279C002F69A5 /* MaterialIcons.ttf */; };
 		4E9EC54A29BA279C002F69A5 /* MaterialIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 4E9EC54629BA279C002F69A5 /* MaterialIcons.ttf */; };
 		4EBCD5B22971514E00D3C04C /* Foundation.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 4EBCD5B12971514E00D3C04C /* Foundation.ttf */; };
-		51A6E1C8BEDF3FA779731DCF /* BuildFile in Frameworks */ = {isa = PBXBuildFile; };
+		51A6E1C8BEDF3FA779731DCF /* (null) in Frameworks */ = {isa = PBXBuildFile; };
```

Comment:
I'm not sure why this and [this](https://github.com/ChildMindInstitute/mindlogger-app-refactor/blob/a94b95d14801dcc90ebf841b16dfd2cf9fdc1616/ios/MindloggerMobile.xcodeproj/project.pbxproj#L341) changed to null. @BamMironov do you have any idea?

Reply:
Hmm, that is not the first time I see this.
Do you use [bundler](https://tingyishih.medium.com/why-bundle-exec-whats-the-difference-ae7ba488a324)? 
If not, please give it a try/
I usually run it with "yarn pods" command.

Reply:
Yup, that did it. Thanks 🙏